my_gspread.py

```diff
@@ -1,23 +1,59 @@
 import gspread
 import pandas as pd
+from pandas import json_normalize
 from dotenv import load_dotenv
 import os 
 import logging
+import requests
+import json
+import datetime as dt
 
+# access keys using dotenv
 load_dotenv()
-API_key =  os.getenv('Project_key')
+gspread_key =  os.getenv('Project_key')
```

Comment:
Use upper case for your environment variable names e.g PROJECT_KEY

Reply:
I will change that now sir